server/server.js

```diff
@@ -1,150 +1,159 @@
-// EXPRESS DEPENDENCIES
 const express = require('express');
 const path = require('path');
 const bodyParser = require('body-parser');
 const morgan = require('morgan');
 const cors = require('cors');
-// require('dotenv').config({ path: path.resolve(__dirname, `../../${process.env.NODE_ENV}.env`) });
-// DATABASE DEPENDENCY
 
 // super test for server - fernando
 
+
 const database = 'postgres';
 
-const HOST_PORT = 3000;
 const app = express();
+const port = 8080;
+
 
 app.use(cors());
-app.use(express.static(path.join(__dirname, '../public')));
 app.use(morgan('dev'));
 app.use(bodyParser.json());
+app.use(express.static(path.join(__dirname, '../public')));
 
-app.get('/:projectId', (req, res) => {
-  res.sendFile(path.resolve(__dirname, '../public/index.html'));
-});
+// app.get('/:projectId', (req, res) => {
+//   res.sendFile(path.resolve(__dirname, '../public/index.html'));
+// });
 
 if (database === 'postgres') {
 
-  const initializeSequelize = require('../database/db.js');
-  // const db = require('../database/index.js');
-
-  app.get('/:projectId/updates', (request, response) => {
-    const { Update, sequelizeConnection, sequelize } = initializeSequelize();
-    sequelizeConnection.then(() =>
-      Update.findAll({
-        where: {
-          projectId: request.params.projectId
-        }
-      })
-        .then(updates => response.send(updates))
-        .then(() => sequelize.close())
-        .catch(err => console.error(err))
-    );
+  const db = require('../database/postgres/index.js');
+
+
+  app.get('/api/:projectId/updates', (request, response) => {
+    const { projectId } = request.params;
+    // const projectId = request.params.projectId;
+    db.findUpdates(projectId, (error, results) => {
```

Comment:
There doesn't appear to be any input validation for the request parameters. You should verify the ID is exactly the right type (which for now is probably a number but later could be a formatted string). You would want to do this so you can respond with error code 400.